woocommerce/woocommerce
# add: customize store design with ai loader

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





- Implements the design for the Loader in the Design With AI wizard
- Design can be found in this issue: https://github.com/woocommerce/woocommerce/issues/39712
- Decided to split this out into a small task that can be quickly completed and reviewed because it's Friday 😄 
- Fixed the stories for the other design with AI pages



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to `tools/storybook` and run `pnpm run storybook`
2. Should find the loader as below:

https://github.com/woocommerce/woocommerce/assets/27843274/d6aa7429-8dbb-4304-bc37-5c29f3a16fea




### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
